qemu/tests/cfg/virtio_fs_guest_users_allowed_by_group_permission.cfg

```diff
@@ -0,0 +1,42 @@
+- virtio_fs_guest_users_allowed_by_group_permission:
+    type = virtio_fs_guest_users_allowed_by_group_permission
```

Comment:
And suggest updating the case name with less characters.
How about " virtio_fs_supp_group_transfer"